pkg/vm/ch.go

```diff
@@ -17,9 +19,34 @@ import (
 )
 
 const (
-	chBin = "cloud-hypervisor"
+	chBin           = "cloud-hypervisor"
+	cloudConsoleBin = "cloud-console"
 )
 
+func (m *Machine) StartCloudConsole(ctx context.Context, namespace string, networkAddr net.IPNet, machineIP net.IPNet, ptyPath string) error {
+	machineIPLsit := strings.Split(machineIP.IP.String(), ".")
```

Comment:
machineIPLsit is a typo, not sure if you mean the ipOctets that are separated by dots if so please use ipOctets instead?